content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -192,22 +192,33 @@ The following flags can be used:
 You can choose more than one authorization module. Modules are checked in order
 so an earlier module has higher priority to allow or deny a request.
 
-## Privilege escalation via pod creation
+## Privilege escalation via workload creation or edits {#privilege-escalation-via-pod-creation}
 
-Users who have the ability to create pods in a namespace can potentially
-escalate their privileges within that namespace.  They can create pods that
-access their privileges within that namespace. They can create pods that access
-secrets the user cannot themselves read, or that run under a service account
-with different/greater permissions.
+Users who can create/edit pods in a namespace, either directly or through a [controller](/docs/concepts/architecture/controller/)
+such as an operator, could escalate their privileges in that namespace.
```

Comment:
```suggestion
such as an operator or deployment, could escalate their privileges in that namespace.
```

Reply:
@geoffcline how does this sound? I want to make it clear that editing the objects which create pods can also be used for this.

Reply:
great!